src/main/java/interfaceadapters/CreateCourseInterfaceAdapters/SuccessCreate.java

```diff
@@ -0,0 +1,9 @@
+package interfaceadapters.CreateCourseInterfaceAdapters;
+
+/**
+ * Exception raised when user successfully created courses.
+ */
+public class SuccessCreate extends RuntimeException{
+    public SuccessCreate(String s) { super(s);
+    }
+}
```

Comment:
I think this class is not required. Because exceptions are only ever attributed to "exceptional failures" that the code is hoping to catch. A success creation is what we expect, so we shouldn't throw an error. This class isn't used anywhere in the code so removing it wouldn't cause any issues.